runners/google-cloud-dataflow-java/src/main/java/org/apache/beam/runners/dataflow/options/DataflowPipelineOptions.java

```diff
@@ -61,14 +67,15 @@
    *
    * <p>Must be a valid Cloud Storage URL, beginning with the prefix "gs://"
    *
-   * <p>At least one of {@link PipelineOptions#getTempLocation()} or {@link #getStagingLocation()}
+   * <p>At least one of {@link GcpOptions#getGcpTempLocation()} or {@link #getStagingLocation()}
```

Comment:
I think the real requirement now is that `gcpTempLocation` must be set, and `stagignLocation` can optionally also be set.

@francesperry FYI